src/PluginPage.tsx

```diff
@@ -0,0 +1,7 @@
+import { PluginPageProps, PluginPage as RealPluginPage, config } from "@grafana/runtime";
```

Comment:
I'm not sure how you managed to get this to build Torkel. The version of `runtime` in declared in `package.json` doesn't have these imports available. Should we bump the `@grafana` package versions to 9.1.x? Not sure how else we're going to get this to work. 🤔 

```
  ERROR in /Users/jackwestbrook/Projects/grafana-plugins/grafana-starter-app/src/pages/CanvasPage.tsx
  ERROR in /Users/jackwestbrook/Projects/grafana-plugins/grafana-starter-app/src/pages/CanvasPage.tsx(2,25):
  TS2305: Module '"../../node_modules/@grafana/runtime"' has no exported member 'PluginPageLayout'.

  ERROR in /Users/jackwestbrook/Projects/grafana-plugins/grafana-starter-app/src/PluginPage.tsx
  ERROR in /Users/jackwestbrook/Projects/grafana-plugins/grafana-starter-app/src/PluginPage.tsx(1,10):
  TS2305: Module '"../node_modules/@grafana/runtime"' has no exported member 'PluginPageProps'.

  ERROR in /Users/jackwestbrook/Projects/grafana-plugins/grafana-starter-app/src/PluginPage.tsx
  ERROR in /Users/jackwestbrook/Projects/grafana-plugins/grafana-starter-app/src/PluginPage.tsx(1,27):
  TS2305: Module '"../node_modules/@grafana/runtime"' has no exported member 'PluginPage'.
```

Reply:
@jackw you need to yarn link things :)

Reply:
@jackw is there a way to specify in package.json to reference canary (main) builds of npm packages?

Reply:
Ah of course. So we could do `"@grafana/runtime": "canary"` but it's still gonna add a "specific" version to the lock file any how. Might be nicer to go with the last canary release `9.2.0-79773pre` for now so it's obvious what version we're using.